react_rails_getting_started.org

```diff
@@ -333,9 +315,11 @@ class HomeController < ApplicationController
 end
 #+END_SRC
 
-****I want to note this is what I expect to happen, and sometimes it happens, sometimes the components get resolved differently - asking Mitch about it ****
+#+BEGIN_QUOTE
+/I want to note this is what I expect to happen, and sometimes it happens, sometimes the components get resolved differently - asking Mitch about it./
+#+END_QUOTE
 
```

Comment:
At this time, I'm pretty lost about everything going on below until the end of this section.